packages/front-end/tests/e2e/lab-technician/Labs_Lab_Technician.spec.e2e.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from 'playwright/test';
+
+test('01 - Create a Laboratory Unsuccessfully', async ({ page, baseURL }) => {
```

Comment:
Should this read "Successfully" instead of "Unsuccessfully"?

Reply:
This one is intentionally 'unsuccessfully' as it only checks the create lab button whether existing